### PR TITLE
evaluate castType.getKind() only once and then compare many times

### PR DESCRIPTION
**Pull request by smonteiro**

### CODE REVIEW

File: framework/src/main/java/org/checkerframework/common/value/ValueVisitor.java [hunk 1]
```diff
@@ -29,7 +29,7 @@
 import org.checkerframework.framework.type.AnnotatedTypeMirror;
 import org.checkerframework.framework.type.AnnotatedTypeMirror.AnnotatedDeclaredType;
 import org.checkerframework.framework.type.visitor.AnnotatedTypeScanner;
-import org.checkerframework.javacutil.AnnotationUtils;
+import static org.checkerframework.javacutil.AnnotationUtils.*;
```
Comment (wmdietl) [inline framework/src/main/java/org/checkerframework/common/value/ValueVisitor.java]: In general we avoid wildcard imports and there are very few static imports.
Please only import the methods you actually use.

Comment (smonteiro) [reply framework/src/main/java/org/checkerframework/common/value/ValueVisitor.java]: Can do. Next commit will have it split.

**Comment by smonteiro** [comment]
This is as far as I want to  go with this pull request. No more commits.

**Comment by mernst** [comment]
As a general rule, we like to make every pull request a logical unit.  This one combines unrelated changes:  using static imports, and avoiding recomputation of `getKind`.  I will merge it, but in the future could you make a separate pull request for each?  I realize that's a touch more work, but it makes each pull request (and commit in the history) easier to understand.
Thanks again.

**Comment by smonteiro** [comment]
I agree completely. I have no issue complying with that as that is what I request also when I do reviews for merge :)
I appreciate the flexibility while I get my "programming legs" with the cadence. 

About the gradlew reformat with Windows: should I file that as an issue in this repository with details or what do you recommend ?

**Pull request merged**